packages/builders/src/cypress/cypress.builder.ts

```diff
@@ -155,7 +155,7 @@ export default class CypressBuilder implements Builder<CypressBuilderOptions> {
   private copyCypressFixtures(tsConfigPath: string, cypressConfigPath: string) {
     const cypressConfig = JSON.parse(readFile(cypressConfigPath));
     // DOn't copy fixtures if cypress config does not have it set
-    if (!cypressConfig.fixtures) {
+    if (!cypressConfig.fixturesFolder) {
       return;
     }
 
```

Comment:
Can you please also make the change on https://github.com/nrwl/nx/pull/1053/files#diff-d2e9d50fe01d1477d617534396da567fR164